src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md

```diff
@@ -0,0 +1,21 @@
+---
+title: 'New Relic now supports native log forwarding from Azure and GCP'
+summary: 'Start monitoring logs alongside your cloud infrastructure metrics today without the need for unsupported forwarders or custom solutions'
+releaseDate: '2022-03-3'
+learnMoreLink: 'https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent'
+getStartedLink: 'https://one.newrelic.com/launcher/logger.log-launcher'
+---
+All existing New Relic accounts using Azure or GCP cloud infrastructure integrations can now send logs to New Relic One without the need for unsupported forwarders or custom solutions published by customer success teams.
+
+Sending logs to New Relic One is a great way to get more detailed and comprehensive information about the services and instances running on your cloud providers. Monitoring logs alongside metrics make it easier to troubleshoot and identify the root cause of issues faster.
+
+New logs integrations:
+
+* **For Azure**, Blob Storage, or via an Event Hub.
+* **For GCP**, pub/sub integration.
+
+!["Azure and CGP Integration Icons"](./images/azure_gcp_logs_icons.png "Azure and CGP Integration Icons")
+
+<figcaption>Azure and GCP log integrations are part of New Relic’s log forwarding plugins that make it easy for our customers to get logs into New Relic One.</figcaption>
+
+To start using logs with Azure or GCP, or any of New Relic’s native log integrations simply type *‘logs’* by clicking on [**+ Add More Data**](https://one.newrelic.com/nr1-core?account=[]&state=7db6a5b5-d270-7c21-d96e-af62e2a6c95f) on your NR1 homepage.
```

Comment:
New Relic One should be spelled out (not NR1)

Reply:
Noted, will be corrected once the PR goes live.

Reply:
@alexiskjones, I revised these instructions slightly by just asking users to click the link, which seems to handle everything.

```
To start using logs with Azure or GCP, or any of New Relic’s native log integrations:

1. Click this [**+ Add More Data**](https://one.newrelic.com/nr1-core?account=[]&state=7db6a5b5-d270-7c21-d96e-af62e2a6c95f) link.
2. In the search bar, type **logs**, and press Enter to see your options.

```